--- rrdtool-1.3.4/bindings/tcl/tclrrd.c	2008-06-08 08:55:09.000000000 -0700
+++ rrdtool-1.3.4/bindings/tcl/tclrrd.c.new	2008-12-03 14:14:53.000000000 -0800
@@ -645,7 +645,7 @@
     if (Tcl_InitStubs(interp, TCL_VERSION, 0) == NULL)
         return TCL_ERROR;
 
-    if (Tcl_PkgRequire(interp, "Tcl", TCL_VERSION, 1) == NULL) {
+    if (Tcl_PkgRequire(interp, "Tcl", TCL_VERSION, 0) == NULL) {
         return TCL_ERROR;
     }
 
