diff -Naurp rrdtool-1.5.5/bindings/tcl/Makefile.am rrdtool-1.5.5.oden/bindings/tcl/Makefile.am
--- rrdtool-1.5.5/bindings/tcl/Makefile.am	2015-11-10 16:07:11.000000000 +0100
+++ rrdtool-1.5.5.oden/bindings/tcl/Makefile.am	2015-11-16 09:36:10.825272050 +0100
@@ -14,21 +14,16 @@ TCL_PACKAGE_PATH = @TCL_PACKAGE_PATH@
 TCL_LD_SEARCH_FLAGS = @TCL_LD_SEARCH_FLAGS@
 TCL_STUB_LIB_SPEC = @TCL_STUB_LIB_SPEC@
 TCL_INCLUDE_SPEC = @TCL_INCLUDE_SPEC@
+TCL_VERSION = @TCL_VERSION@
 
 SRC_DIR            = $(top_srcdir)/src
 AM_CPPFLAGS        = $(TCL_INCLUDE_SPEC) -I$(top_builddir)/src -I$(SRC_DIR) -DUSE_TCL_STUBS
 LIBDIRS            = -L$(top_builddir)/src/.libs -L$(top_builddir)/src -L$(libdir)
 LIB_RUNTIME_DIR    = $(libdir)
 
-if BUILD_TCL_SITE
-tclpkgdir = @TCL_PACKAGE_DIR@
+tclpkgdir = $(libdir)/tcl$(TCL_VERSION)/tclrrd
 tclpkg_DATA = pkgIndex.tcl
 tclpkg_SCRIPTS = ifOctets.tcl
-else
-pkgindexdir = $(pkglibdir)
-pkgindex_DATA = pkgIndex.tcl
-pkgindex_SCRIPTS = ifOctets.tcl
-endif
 
 # Automake doesn't like `tclrrd$(VERSION)$(TCL_SHLIB_SUFFIX)' as
 # library name. So we build and install this library `by hand'.
