diff -Naurp rrdtool-1.5.5/bindings/lua/Makefile.am rrdtool-1.5.5.oden/bindings/lua/Makefile.am
--- rrdtool-1.5.5/bindings/lua/Makefile.am	2015-04-20 16:12:18.000000000 +0200
+++ rrdtool-1.5.5.oden/bindings/lua/Makefile.am	2015-11-16 09:33:05.213913170 +0100
@@ -26,7 +26,7 @@ else
   rrd_la_SOURCES = rrdlua.c
 endif
 rrd_la_LIBADD = -lrrd -lm
-rrd_la_LDFLAGS = -module -version-info $(LIB_VERSION_INFO) $(LUA_LFLAGS) -L$(top_builddir)/src/.libs
+rrd_la_LDFLAGS = -module -avoid-version $(LUA_LFLAGS) -L$(top_builddir)/src/.libs
 rrd_la_CFLAGS = -I$(top_srcdir)/src -I$(top_builddir)/src -I$(top_srcdir) $(LUA_CFLAGS) $(LUA_DEFINES) -DLIB_VERSION=\"$(LIB_VERSION)\"
 
 .PHONY: test.lua
