diff -Naurp rrdtool-1.5.5/examples/4charts.pl.in rrdtool-1.5.5.oden/examples/4charts.pl.in
--- rrdtool-1.5.5/examples/4charts.pl.in	2015-11-10 17:06:16.000000000 +0100
+++ rrdtool-1.5.5.oden/examples/4charts.pl.in	2015-11-16 09:28:53.306630853 +0100
@@ -1,8 +1,5 @@
 #! @PERL@
 
-#makes things work when run without install
-use lib qw( @prefix@/lib/perl );
-
 use RRDs;
 
 my $start=time;
diff -Naurp rrdtool-1.5.5/examples/bigtops.pl.in rrdtool-1.5.5.oden/examples/bigtops.pl.in
--- rrdtool-1.5.5/examples/bigtops.pl.in	2015-11-10 17:06:16.000000000 +0100
+++ rrdtool-1.5.5.oden/examples/bigtops.pl.in	2015-11-16 09:28:53.306630853 +0100
@@ -1,6 +1,4 @@
 #! @PERL@
-# this is for after install
-use lib qw( @prefix@/lib/perl );
 
 use RRDs;
 my $start=time;
diff -Naurp rrdtool-1.5.5/examples/minmax.pl.in rrdtool-1.5.5.oden/examples/minmax.pl.in
--- rrdtool-1.5.5/examples/minmax.pl.in	2015-11-10 17:06:16.000000000 +0100
+++ rrdtool-1.5.5.oden/examples/minmax.pl.in	2015-11-16 09:28:53.306630853 +0100
@@ -1,7 +1,5 @@
 #! @PERL@
 
-use lib qw( @prefix@/lib/perl );
-
 use RRDs;
 my $start=time;
 my $rrd="randome.rrd";
diff -Naurp rrdtool-1.5.5/examples/perftest.pl.in rrdtool-1.5.5.oden/examples/perftest.pl.in
--- rrdtool-1.5.5/examples/perftest.pl.in	2012-09-11 14:44:52.000000000 +0200
+++ rrdtool-1.5.5.oden/examples/perftest.pl.in	2015-11-16 09:28:53.306630853 +0100
@@ -5,12 +5,9 @@
 # Created By Tobi Oetiker <tobi@oetiker.ch>
 # Date 2006-10-27
 #
-#makes program work AFTER install
 
 my $Chunk = shift @ARGV || 10000;
 
-use lib qw( ../bindings/perl-shared/blib/lib ../bindings/perl-shared/blib/arch @prefix@/lib/perl );
-
 print <<NOTE;
 
 RRDtool Performance Tester
diff -Naurp rrdtool-1.5.5/examples/piped-demo.pl.in rrdtool-1.5.5.oden/examples/piped-demo.pl.in
--- rrdtool-1.5.5/examples/piped-demo.pl.in	2012-09-11 11:07:00.000000000 +0200
+++ rrdtool-1.5.5.oden/examples/piped-demo.pl.in	2015-11-16 09:28:53.307630828 +0100
@@ -1,7 +1,5 @@
 #! @PERL@ 
 
-use lib qw( @prefix@/lib/perl );
-
 use RRDp;
 
 # this simulates a standard mrtg-2.x setup ... we can use this to
diff -Naurp rrdtool-1.5.5/examples/shared-demo.pl.in rrdtool-1.5.5.oden/examples/shared-demo.pl.in
--- rrdtool-1.5.5/examples/shared-demo.pl.in	2015-11-10 17:06:16.000000000 +0100
+++ rrdtool-1.5.5.oden/examples/shared-demo.pl.in	2015-11-16 09:28:53.307630828 +0100
@@ -14,9 +14,6 @@ sub ok
     print "ok $ok_count $what\n";
 }
 
-#makes program work AFTER install
-use lib qw( @prefix@/lib/perl );
-
 use strict;
 use vars qw(@ISA $loaded);
 
diff -Naurp rrdtool-1.5.5/examples/stripes.pl.in rrdtool-1.5.5.oden/examples/stripes.pl.in
--- rrdtool-1.5.5/examples/stripes.pl.in	2015-11-10 17:06:16.000000000 +0100
+++ rrdtool-1.5.5.oden/examples/stripes.pl.in	2015-11-16 09:28:53.307630828 +0100
@@ -1,5 +1,4 @@
 #! @PERL@
-use lib qw( @prefix@/lib/perl );
 
 use strict;
 use vars qw(@ISA $loaded);
