diff --git a/bindings/tcl/Makefile.am b/bindings/tcl/Makefile.am
index 721580b..903a09b 100644
--- a/bindings/tcl/Makefile.am
+++ b/bindings/tcl/Makefile.am
@@ -14,23 +14,17 @@ TCL_PACKAGE_PATH = @TCL_PACKAGE_PATH@
 TCL_LD_SEARCH_FLAGS = @TCL_LD_SEARCH_FLAGS@
 TCL_STUB_LIB_SPEC = @TCL_STUB_LIB_SPEC@
 TCL_INCLUDE_SPEC = @TCL_INCLUDE_SPEC@
+TCL_VERSION = @TCL_VERSION@
 
 SRC_DIR            = $(top_srcdir)/src
 AM_CPPFLAGS        = $(TCL_INCLUDE_SPEC) -I$(top_builddir)/src -I$(SRC_DIR) -DUSE_TCL_STUBS
 LIBDIRS            = -L$(top_builddir)/src/.libs -L$(top_builddir)/src -L$(libdir)
 LIB_RUNTIME_DIR    = $(libdir)
 
-if BUILD_TCL_SITE
-tclpkgdir = @TCL_PACKAGE_DIR@
+tclpkgdir = $(libdir)/tcl$(TCL_VERSION)/tclrrd
 tclpkg_DATA = pkgIndex.tcl
 tclpkg_SCRIPTS = ifOctets.tcl
 tclpkg_rellibdir = $(libdir)
-else
-tclpkgdir = $(pkglibdir)
-tclpkg_DATA = pkgIndex.tcl
-tclpkg_SCRIPTS = ifOctets.tcl
-tclpkg_rellibdir = ..
-endif
 
 # Automake doesn't like `tclrrd$(VERSION)$(TCL_SHLIB_SUFFIX)' as
 # library name. So we build and install this library `by hand'.
